feature_layers/display-route-layer/README.metadata.json

```
@@ -0,0 +1,36 @@
+{
+    "category": "Feature Layers",
+    "description": "Display a route layer and its directions using a feature collection.",
+    "ignore": false,
+    "images": [
+        "DisplayRouteLayer.png"
+    ],
+    "keywords": [
+        "collection",
+        "feature collection",
+        "feature collection layer",
+        "id",
+        "item",
+        "map notes",
+        "portal",
+        "Feature",
+        "FeatureCollection",
+        "FeatureCollectionLayer",
+        "Portal",
+        "PortalItem"
+    ],
+    "redirect_from": [
+        "/java/latest/sample-code/display-a-map.htm"
+    ],
```

```suggestion
    "redirect_from": [],
```
We don't need a redirect here

---

removed extra code from here